### PR TITLE
Documentation for Elasticsearch usage in ALT

### LINKED ISSUE #292: Elasticsearch documentation
- document specifics for ALT regarding ES

### PR DESCRIPTION
**Pull request by DanaSunal**
Fixes #292

### CODE REVIEW

**Comment by DanaSunal** [comment]
Reviewer is @IswaryaaS

**Review by IswaryaaS — CHANGES_REQUESTED** [review]
1. Requesting you to please create an issue for including documentation and link this PR to the issue. 
2. we need to include the main.md file in /doc/ folder and use this as root file to include all other concepts. 
Please refer to https://github.com/openBackhaul/RegistryOffice/blob/develop/doc
3. After including main.md file, we need to refer it in [Description](https://github.com/DanaSunal/ApplicationLayerTopology/tree/docs#description) and [Resources](https://github.com/DanaSunal/ApplicationLayerTopology/tree/docs#resources) in Readme.md . Please refer to https://github.com/openBackhaul/RegistryOffice#readme

**Review by IswaryaaS — APPROVED** [review]

**Review by IswaryaaS — COMMENTED** [review]
Requesting second level review from @vanithavalluripalli9

File: doc/ElasticsearchUsage.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+## Using Elasticsearch in ApplicationLayerTopology Application
+
```
Comment (IswaryaaS) [inline doc/ElasticsearchUsage.md:2]: I understand for comment #4 of https://github.com/openBackhaul/ApplicationLayerTopology/pull/284#pullrequestreview-1378619863 has been missed to address. 
"Also requesting you to include reference of ES documents from ApplicationPattern repository, so when a new user starts with ALT, he can go to AP documents to understand the basic or any further information required.", we could include the link after it is merged to AP.

**Review by IswaryaaS — CHANGES_REQUESTED** [review]
Also in readme.md file, along with adding url to resources column, we need to include reference to main.md in [Description](https://github.com/DanaSunal/ApplicationLayerTopology/tree/docs#description) similar to https://github.com/openBackhaul/RegistryOffice#description

File: doc/ElasticsearchUsage.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+};
+```
+```
+module.exports.putElasticsearchClientServiceRecordsPolicy = function putElasticsearchClientServiceRecordsPolicy (req, res, next, body, uuid) {
+  let responseCode = responseCodeEnum.code.METHOD_NOT_ALLOWED;
+  responseBuilder.buildResponse(res, responseCode, {});
+  oamLogService.recordOamRequest(req.url, req.body, responseCode, req.headers.authorization, req.method);
+};
+```
+
```
Comment (vanithavalluripalli9) [inline doc/ElasticsearchUsage.md:25]: can we mention why we are not using service record policy in ALT

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]

File: doc/ElasticsearchUsage.md (hunk 1 above)
Comment (DanaSunal) [reply doc/ElasticsearchUsage.md:2]: Yes,sorry, missed that one. Fixed now.

File: doc/ElasticsearchUsage.md (hunk 2 above)
Comment (DanaSunal) [reply doc/ElasticsearchUsage.md:27]: Done.

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**